reconcile/cli.py

```diff
@@ -1869,6 +1869,30 @@ def template_validator(ctx):
     )
 
 
+@integration.command(short_help="Render datafile templates in app-interface.")
+@click.option(
+    "--app-interface-data-path",
+    help="Path to app-interface dictory, used to write output and calculate diff in dry-run.",
+    required=False,
+    envvar="APP_INTERFACE_DATA_PATH",
+)
+@click.pass_context
+def template_renderer(ctx, app_interface_data_path):
```

Comment:
Is this a long running integration? I thought it's a cli command in `qontract_cli.py`, this `--app-interface-data-path` won't work in real integration run container, right?

Reply:
It will be one in the next iteration. I wanted to make the PRs shorter and easier to review. Think MR management (which will make this a long-running integration) would have made this way more complicated.